### PR TITLE
Some supervisord configuration were missing regarding onboarder documentation

### PR DESCRIPTION
**Pull request by AymericPlanche**
**Description**

A customer had errors when he ran the job queue daemon command line.

According to the official PIM installation procedure this daemon is configured to be ran / supervised by supervisord process.
According to the supervisord official documentation the environment variables of the shell which runs the supervisord main process are not shared with the processes it manage. They have to be explicitly written in the `/etc/supervisor.conf` configuration file.

This PR add this information and add a troubleshooting section to ease the search of this kind of errors through the docs.akeneo.com website.

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: onboarder/synchronization/index.rst [hunk 1]
```diff
@@ -31,13 +31,13 @@ Launch the message worker
 | The messages that are part of the synchronization process are queued.
 | The queue is consumed by a command line process called ``worker``.
 |
-| The worker has to always be launched as it polls the queue waiting for new messages to handle.
+| The worker as to always be launched as it polls the queue waiting for new messages to handle.
```
Comment (damien-carcel) [inline onboarder/synchronization/index.rst]: nope, `has` was good :-)

File: onboarder/synchronization/index.rst [hunk 2]
```diff
@@ -48,6 +48,16 @@ Add the following supervisor configuration (update the following example to your
     autostart=true
     autorestart=true
 
+.. warning::
+
+    By default the environment variable of the shell that is running the supervisord process will not propagate the environment variables to the process it monitor.
+    You've to configure :doc:`the mandatory environment variables </onboarder/installation/index>` that the akeneo/pim-onboarder bundle requires in the ``/etc/supervisor.conf`` file.
```
Comment (damien-carcel) [inline onboarder/synchronization/index.rst]: `You have to`. Looks weird to me to read an abbreviation.

File: onboarder/synchronization/index.rst (hunk 1 above)
Comment (AymericPlanche) [inline onboarder/synchronization/index.rst]:
```suggestion
| The worker has to always be launched as it polls the queue waiting for new messages to handle.
```

File: onboarder/synchronization/index.rst (hunk 2 above)
Comment (AymericPlanche) [inline onboarder/synchronization/index.rst]:
```suggestion
    You have to configure :doc:`the mandatory environment variables </onboarder/installation/index>` that the akeneo/pim-onboarder bundle requires in the ``/etc/supervisor.conf`` file.
```

**Pull request merged**